eval.go

```diff
@@ -391,10 +349,6 @@ nextFileMatch:
 		res.Stats.FileCount++
 	}
 
-	// We do not sort Files here, instead we rely on the shards pkg to do file
-	// ranking. If we sorted now, we would break the assumption that results
-	// from the same repo in a shard appear next to each other.
-
```

Comment:
Is this comment not relevant anymore?

Reply:
Right, it's been superseded by a more helpful comment below when we sort and truncate the scores.